common/persistence/DriverConfigurationFeeder.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ */
+
+declare(strict_types=1);
+
+namespace oat\generis\persistence;
+
+use oat\oatbox\service\ConfigurableService;
+
+class DriverConfigurationFeeder extends ConfigurableService
```

Comment:
I do not get the value of this class, could we just declare a spannerDriver in the persistence.conf.php. As far as I remember it is the purpose of this library driver, isn't it?

Reply:
Can you show me how you would to it with the needs I have here? I did not find any way of doing that.